tests/src/JIT/SIMD/AbsSqrt.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Collections.Generic;
+using System.Numerics;
+using Point = System.Numerics.Vector4;
+
+namespace VectorMathTests
+{
+    class Program
+    {
+
+
+
+        static int Main(string[] args)
+        {
+            Point a = new Point(1, 2, 3, 4);
+            Point b = new Point(2, 2, 1, 1);
+            float c = 33;
+            Point d = (b + a) * c;
+            Point q = d + a;
+            Point s = Point.SquareRoot(q);
+            s *= -1;
+            s = Point.Abs(s);
+            return (int)(s).X * 10;
```

Comment:
Please don't combine a computation with the Pass/Fail return value - it obscures what is going on. Rather, you should either add a comment that 100 is the expected value of the expression (and compare against that), or assign it to a variable such as "Result" and compare against the expected value. Then, return Pass or Fail.